app/processors/audio_processor.rb

```diff
@@ -120,7 +120,7 @@ def validate_audio
   def cut_audio
     source_wav = get_wav_from_source
     task_tmp = audio_monster.create_temp_file(File.basename(source.path))
-    audio_monster.cut_wav(source_wav.path, task_tmp.path, options['length'], options['fade'] || 5 )
+    audio_monster.cut_wav(source_wav.path, task_tmp.path, options['length'].to_i, (options['fade'] || 5).to_i )
```

Comment:
Line is too long. [111/80]<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.